src/common/obj_bencher.cc

```diff
@@ -153,8 +153,12 @@ void *ObjBencher::status_printer(void *_bencher) {
 
 int ObjBencher::aio_bench(
   int operation, int secondsToRun,
+<<<<<<< HEAD
   int maxObjectsToCreate,
   int concurrentios, int op_size, bool cleanup, const char* run_name) {
+=======
+  int concurrentios, int object_size, bool cleanup, const char* run_name, bool no_verify) {
+>>>>>>> ca6abca... tools: add --no-verify option to rados bench
```

Comment:
I'm assuming the second version of the line ie: the line with bool no verfiy should be the only line present.. line 158 should be deleted